src/shipit_static_analysis/shipit_static_analysis/clang/tidy.py

```diff
@@ -140,16 +142,27 @@ def parse_issues(self, clang_output, revision):
         if not headers:
             raise Exception('No clang-tidy header was found even though a clang output was provided')
 
+        def _parse_body(start_pos, end_pos):
```

Comment:
Maybe `_remove_footer`?

Reply:
We're not only doing that, but we're not really parsing, not sure what a good name would be.